CHANGELOG

```diff
@@ -1,5 +1,68 @@
 This file contains a summary of changes to the Oppia code base.
 
+v2.7.3 (16 February 2019)
+------------------------
+
+Infrastructural changes
+* Improvements to PlaythroughAudit (#6239)
+* Add None/deleted checks to DeleteIllegalPlaythroughsOneOffJob (#6195)
+* Fix Content-Disposition headers being unicode instead of str (#6192)
+* Fix check for clean working tree in common.py (#6191)
+* Fix #4666: Fixes correctness test for Text Input Prediction Service  (#6120)
+* Fix #5975: Create One-off Job to Remove Invalid Playthroughs (#6131)
+* Fix #5371: One Off Job to populate message_count in GeneralFeedbackThreadModel (#5999)
+* Fix #6104: Use sub-process in pre_push_hook (#6114)
+* Fix part of #5591: Migrate Playthrough Feature Data into ExplorationFeaturesService (#6044)
+* Fix part of #5002: Move ALLOW_YAML_FILE_UPLOAD from feconf to constants (#5989)
+* Fix part of #5002: Remove GLOBALS from HTML (#6050)
+* Add audio translation schema for subtopic pages (#5823)
+* Remove automatic backups script. (#5956)
+* Fix classifier models when exploration is reverted and add test case for auto migrations (#5888)
+
+Improvements to the editors and players
+* Fix#6188: Fix word-break in explorations (#6187)
+* Changed word-break to word-wrap (#6189)
+* Fix #5745: Redesign audio bar of the translation tab. (#6167)
+* Fix #5822: Add customization option to allow multiple sort items in the same position in Drag and drop interaction (#6177)
+* UI changes to topic and skill editor (#6091)
+* Fix #5019 : The continue button is now visible with long feedback (#6136)
+* Fix #6122: Add $log service to prevent the error arising on rejecting suggestions (#6124)
+* Fix #4407: KaTeX text no longer goes outside text container in MathExpressionInput (#6130)
+* Fix #4833: prevents text overflow  in answer statistics page (#6137)
```

Comment:
Maybe be consistent about whether the first letter after the colon is capitalized or not